OrbitGl/TimeGraphLayout.cpp

```diff
@@ -36,47 +36,41 @@ TimeGraphLayout::TimeGraphLayout() {
   time_bar_height_ = 15.f;
 };
 
-#define FLOAT_SLIDER_MIN_MAX(x, min, max)     \
-  if (ImGui::SliderFloat(#x, &x, min, max)) { \
-    needs_redraw = true;                      \
+#define IMGUI_FLOAT_SLIDER(x) IMGUI_FLOAT_SLIDER_MIN_MAX(x, 0, 100.f)
+#define IMGUI_FLOAT_SLIDER_MIN_MAX(x, min, max)                                       \
+  if (ImGui::SliderFloat(ImGuiOrbit::PrettyVariableName(#x).c_str(), &x, min, max)) { \
+    needs_redraw = true;                                                              \
   }
 
-#define FLOAT_SLIDER(x) FLOAT_SLIDER_MIN_MAX(x, 0, 100.f)
-
 bool TimeGraphLayout::DrawProperties() {
-  ImGui::SetNextWindowSize(ImVec2(500, 400), ImGuiCond_FirstUseEver);
-  ImVec2 size(400, 400);
-
-  ImGui::Begin("Layout Properties", &m_DrawProperties, size, 1.f, 0);
   bool needs_redraw = false;
-  FLOAT_SLIDER(m_TrackLabelOffsetX);
-  FLOAT_SLIDER(m_TextBoxHeight);
-  FLOAT_SLIDER(m_CoresHeight);
-  FLOAT_SLIDER(thread_state_track_height_);
-  FLOAT_SLIDER(m_EventTrackHeight);
-  FLOAT_SLIDER(m_GraphTrackHeight);
-  FLOAT_SLIDER(m_SpaceBetweenCores);
-  FLOAT_SLIDER(space_between_gpu_depths_);
-  FLOAT_SLIDER(m_SpaceBetweenTracks);
-  FLOAT_SLIDER(m_SpaceBetweenTracksAndThread);
-  FLOAT_SLIDER(m_SpaceBetweenThreadBlocks);
-  FLOAT_SLIDER(m_SliderWidth);
-  FLOAT_SLIDER(time_bar_height_);
-  FLOAT_SLIDER(m_TrackTabHeight);
-  FLOAT_SLIDER(m_TrackTabOffset);
-  FLOAT_SLIDER(m_CollapseButtonOffset);
-  FLOAT_SLIDER(m_RoundingRadius);
-  FLOAT_SLIDER(m_RoundingNumSides);
-  FLOAT_SLIDER(m_TextOffset);
-  FLOAT_SLIDER(right_margin_);
-  FLOAT_SLIDER(m_SchedulerTrackOffset);
-  FLOAT_SLIDER_MIN_MAX(m_TrackTabWidth, 0, 1000.f);
-  FLOAT_SLIDER_MIN_MAX(m_TrackBottomMargin, 0, 20.f);
-  FLOAT_SLIDER_MIN_MAX(m_TrackTopMargin, 0, 20.f);
-  FLOAT_SLIDER(m_ToolbarIconHeight);
-  FLOAT_SLIDER_MIN_MAX(scale_, 0.05f, 20.f);
-  ImGui::Checkbox("DrawTrackBackground", &m_DrawTrackBackground);
-  ImGui::End();
+  IMGUI_FLOAT_SLIDER(m_TrackLabelOffsetX);
```

Comment:
Can you please use this opportunity to update the names here to our style guide?

Reply:
Done.

Reply:
Thanks!